<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkClientClass</class>
 <widget class="QMainWindow" name="NetworkClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetworkClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="my_pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QListWidget" name="my_listWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="my_textEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="my_quitwindow">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionSign_in"/>
    <addaction name="actionSign_up"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuUser"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSign_in">
   <property name="text">
    <string>sign_in</string>
   </property>
  </action>
  <action name="actionSign_up">
   <property name="text">
    <string>sign_up</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NetworkClient.qrc"/>
 </resources>
 <connections/>
</ui>
